frontend/src/scenes/actions/ActionStep.js

```diff
@@ -20,6 +20,14 @@ let getSafeText = (el) => {
 }
 
 export class ActionStep extends Component {
+    isUrlChecked = (step) => {
+        return step?.selection?.indexOf('url') !== -1 ? true : false
```

Comment:
A ternary with `? true : false` is quite odd. `!==` already returns a boolean.

Reply:
done in https://github.com/PostHog/posthog/pull/1680/commits/24bd7e44389826b15f6866a32602820560cbf11b